internal/core/services/room_manager/room_manager_test.go

```diff
@@ -113,7 +113,7 @@ func TestRoomManager_CreateRoom(t *testing.T) {
 
 		roomStorageStatusWatcher.EXPECT().Stop()
```

Comment:
@caiordjesus You forgot to change test func names `TestRoomManager_CreateRoom` and `TestRoomManager_DeleteRoom`